### DISCUSSION TITLE
Cent OS 7 setup script by Dr.Sybren

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2652] [General]
Dr.Sybren just posted a setup script for cent os vm, might be useful later
https://developer.blender.org/P2368

![image](https://user-images.githubusercontent.com/25552173/132335503-fe7bb059-6b62-44ff-ab79-4fa19a7986b5.jpeg)

**Comment by ReinerBforartists** [comment]
Thanks for sharing :)
